ecs-logs-collector.sh

```diff
@@ -172,7 +172,8 @@ try_set_instance_infodir() {
       # Put logs into a directory for this instance.
       infodir="${infodir}/${instance_id}"
       info_system="${infodir}/system"
-      echo "$instance_id" | $info_system/instance-id.txt
+      mkdir -p ${info_system}
```

Comment:
Quick fix: please quote the path here (ie: `mkdir -p "${info_system}"`).

Reply:
This change may make it in with #45, so let's keep an eye on this when rebasing.